pkg/reconciler/testing/service.go

```diff
@@ -14,30 +14,26 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
```

Comment:
lol this is a weird diff

Reply:
/idk

Reply:
I think it is git being funny, it is a new file

Reply:
yeah it's accidentally a rename because git can only detect renames by looking at file similarity.